installing.md

```diff
@@ -0,0 +1,135 @@
+# Installing dependencies natively
+
+Installing the required tools locally takes a bit more work than running through
+singularity, but thanks to python virtual environments it's still quite straight
+forward.
+
+Two tools will be needed for SDA submission:
+
+ - [crypt4gh](https://www.ga4gh.org/news/crypt4gh-a-secure-method-for-sharing-human-genetic-data/)
+   for encrypting files for storage in the secure archive,
+ - [s3cmd](https://s3tools.org/s3cmd) for uploading the encrypted files (or
+   [s3express](https://www.s3express.com/) on windows).
+
+## Installing crypt4gh
+
+The `crypt4gh` tool requires **Python 3.6** or newer, with **python3-venv**.
+
+### Check if Python is installed
+
+To check whether Python is already installed in your machine, you can run the
+following command in the terminal:
+
+```bash
+python3 --version
+```
+
+The output of the command should signify the version of python, in case it is installed, like:
+
+```bash
+    Python 3.6.1
+```
+
+​
+If python isn't installed, or that the version isn't 3.6 or newer, go to the
+[Official python download page](https://www.python.org/downloads/) and follow
+the instructions to install it.
+​
+### Create a virtual environment
+​
+The python virtual environment tools comes bundled with python since version
+3.4, and ​the syntax is the same on all platforms.
+
+```bash
+python3 -m venv venv
+```
+
+This will create a folder called `venv` under the current directory. To activate
+the environment run the following command from the terminal (this command has
+to be run everytime a new terminal window is opened):​
+
+* Linux and macOS
+​
+  ```bash
+  source venv/bin/activate
+  ```
+
+* Windows
+
+  ```PowerShell
+  .\venv\Scripts\activate
+  ```
+
+  **Note** PowerShell doesn't normally allow script execution, and you may need
+  to run
+  ```PowerShell
+  Set-ExecutionPolicy -Scope CurrentUser -ExecutionPolicy RemoteSigned
+  ```
+  before you can activate the virtual environment.
+
+### Installing crypt4gh
+​
+The next step is to install [crypt4gh](https://github.com/EGA-archive/crypt4gh),
+by running the following command:
+​
+```bash
+pip install crypt4gh
+```
+​
+To verify the installation run:
+​
+```bash
+crypt4gh --version
+```
+​
+which should return something like:
+​
+```bash
+    GA4GH cryptographic utilities (version 1.4)
+```
+
```

Comment:
Might add a section about the golang verison of [Crypt4GH](https://github.com/elixir-oslo/crypt4gh)

Reply:
Good idea! :) I'll add that tomorrow!